src/partials/header-content.hbs

```diff
@@ -189,6 +189,7 @@
                 <li><a href="/docs/cypher-manual/current/" class="project-link">Cypher</a></li>
                 <li><a href="/docs/cypher-cheat-sheet/current/" class="project-link">Cypher Cheat Sheet</a></li>
                 <li><a href="/docs/cypher-refcard/current/" class="project-link">Cypher Refcard</a></li>
+                <li><a href="/docs/apoc/current/" class="project-link">APOC User Guide</a></li>
```

Comment:
I was going to suggest that the text here should just be **APOC** but then the docs home page lists it as **APOC Library** in the list on the left. Now I'm unsure what we are calling these docs.

Reply:
I think just APOC makes sense. Will fix.

Reply:
That works for me